overrides.css

```diff
@@ -63,8 +63,8 @@ that are generated by deno_doc, the tool that powers the API reference documenta
   @apply ml-1 !important;
 }
 
-.doc .toc :not(.documentNavigation) > a {
-  @apply text-xl !important;
+.ddoc .z-50 {
+  z-index: 49 !important;
```

Comment:
sorry for this hack, I can't edit the z-50 on the reference doc that is generated

Reply:
Can you isolate it instead? May be a little cleaner. https://tailwindcss.com/docs/isolation